package.json

```diff
@@ -88,7 +88,8 @@
     "react-focus-lock": "^2.9.1",
     "react-portal": "^4.2.1",
     "react-uid": "^2.3.1",
-    "svg4everybody": "^2.1.9"
+    "svg4everybody": "^2.1.9",
+    "yarn": "^1.22.19"
```

Comment:
These changes shouldn't have any package changes. 
Yarn should be installed globally with `npm install --global yarn`
Can you remove this line, delete `package-lock.json` and revert changes to `yarn.lock`?

Reply:
Thanks ,reverted the package.json and changes to yarn.lock

Reply:
I think I'm still seeing changes to `yarn.lock` 
![image](https://user-images.githubusercontent.com/86632227/186272639-56892085-341f-4df4-84c3-f15d4020d946.png)